operator/src/main/resources/application.properties

```diff
@@ -41,12 +41,21 @@ ingresscontroller.max-ingress-connections=108000
 ingresscontroller.peak-connection-percentage=100
 
 # Enables haproxy option contstats in the kas ingress so that stats are reported live rather than at connection close (RFE-3007 provide a proper mechanism to enable this option).
-ingresscontroller.ingress-container-command=/usr/bin/bash,-c,awk '{print $0} /^defaults$/ {print \"  option contstats\"}' < $TEMPLATE_FILE > /tmp/haproxy-config.template; exec /usr/bin/openshift-router --v=2 --template /tmp/haproxy-config.template
+ingresscontroller.ingress-container-command=/usr/bin/bash,-c,awk '{print $0} /^defaults$/ {print \"  option contstats\"}' < $TEMPLATE_FILE > /tmp/haproxy-config.template; exec /usr/bin/openshift-router --v=${ingresscontroller.router-verbosity} --template /tmp/haproxy-config.template --commit-interval=${ingresscontroller.commit-interval} --blueprint-route-pool-size=${ingresscontroller.blueprint-route-pool-size} --max-dynamic-servers=${ingresscontroller.max-dynamic-servers} --blueprint-route-namespace=${ingresscontroller.blueprint-namespace}
 # Disconnect established connections after a haproxy reconfiguration (NE-879 should eliminate the need for this)
 ingresscontroller.hard-stop-after=5s
 # Coalesce up-to reloadInterval worth of haproxy updates.  This prevents a flurry of haproxy restarts (1 per OpenShift Route) as each kafka
 # instance is created (NE-879 should eliminate the need for this).
 ingresscontroller.reload-interval-seconds=60
+ingresscontroller.router-verbosity=2
+ingresscontroller.blueprint-namespace=kas-route-blueprints
+
+# HA proxy dynamic config manager
+ingresscontroller.dynamic-config-manager=true
```

Comment:
Ideally we'd create the blueprints first, then roll the ingresscontrollers, but it's probably not worth accounting for - at worst it seems like we'd end up with a couple of reloads in quick succession and then be stable after that

Reply:
yeah - that was my first thought too.  It turned out to be hard for the reason I give below.  In the algorithm I'm using we will see one reload when it creates the blueprints for the first time.